infra/src/test/kotlin/fi/fta/geoviite/infra/split/SplitTestDataService.kt

```diff
@@ -42,21 +40,17 @@ class SplitTestDataService @Autowired constructor(
 ) : DBTestBase() {
 
     fun insertSplit(
```

Comment:
En lähtenyt siirtämään näitä nyt vielä, mutta sekin voisi olla järkevää. Nämä tekee nyt itsenäisesti päätöksiä siitä että mihin kontekstiin olioita luodaan - voisi olla fiksumpaa tehdä esim tuo switchAndGeometry haluamansa kontekstin kautta ja kaikki luodaan sinne.